src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -120,7 +120,7 @@ class CrossDeviceLinkUI extends Component {
 
       // move focus away from Copy button to prevent screen readers announcing
       // text changing back from "Copied" to "Copy"
-      this.linkText.focus()
+      this.linkText && this.linkText.focus()
```

Comment:
Thanks for catching this! Thought I'd fixed it in one of my earlier PRs, but clearly not. 😅